package.json

```diff
@@ -64,17 +69,13 @@
     "query-string": "^2.4.1",
     "randgen": "^0.1.0",
     "react": "0.13.3",
-    "should": "^7.1.0",
     "sprintf": "^0.1.5",
-    "topojson": "^1.6.19",
```

Comment:
`topojson`is used for pregenerating maps, but it is also needed on the client side, thus not just a regular dependency. We use topojson as described here http://bost.ocks.org/mike/map/